templates/cv_entry.md

```diff
@@ -1,44 +1,53 @@
-<!-- TODO: Create the heading Pt(111) from metadata. See #31. -->
 <!-- TODO: Create the small heading subtitle from metadata. See #31. -->
-# Pt(111) <small>- 0.1 M CsF + 0.1 M HClO4</small>
+# {{ entry.electrochemical_system.electrodes.working_electrode.material }}({{ entry.electrochemical_system.electrodes.working_electrode.crystallographic_orientation }}) <small>- 0.1 M CsF + 0.1 M HClO4</small>
 
 <!-- TODO: Create text from metadata. See #31. -->
 <!-- TODO: We should merge all our .bib files when building the pages and then create reference to original article from that .bib file. See #31. -->
-A cyclic voltammogramm for Pt(111) recorded in 0.1 M CsF + 0.1 M HClO4 at a scan rate of 50 mV s$^{-1}$ from Figure 2a in [N. Author *et al.*, *Journal Name*, **Volume** (YEAR) Page, "TITLE"](https://doi.org/10.1039/C0CP01001D).
+A cyclic voltammogramm for 
+{{ entry.electrochemical_system.electrodes.working_electrode.material }}
+({{ entry.electrochemical_system.electrodes.working_electrode.crystallographic_orientation }}) 
+recorded in 
+0.1 M CsF + 0.1 M HClO4 
+at a scan rate of 
+50 mV s$^{-1}$ <!-- TODO: Create nice representation of the scan rate unit from entry.figure_description.scan_rate_unit -->
+from Figure 
+{{entry.source.figure }} 
+in 
+[N. Author *et al.*, *Journal Name*, **Volume** (YEAR) Page, "TITLE"](https://doi.org/10.1039/C0CP01001D).
 
 <!-- TODO: Show plots with original axis units, see #25. It would be great if we could toggle between SI and original units. See #31. -->
 <!-- TODO: Properly format plots. They should probably be much bigger since they are nice to look at. See #31. -->
 {{ entry.plot()._repr_html_() }}
 
 {{ entry.plot('A / m^2')._repr_html_() }}
 
+**Figure notes:**  
+The figure shows {{ entry.figure_description.type }} data.
+{% if entry.figure_description.comment is defined %}
+{% if entry.figure_description.comment|wordcount() > 1 %}
+Note from the curator: {{ entry.figure_description.comment }}
+{% endif %}
+{% endif %}
```

Comment:
```suggestion
{% if entry.figure_description.comment %}
Note from the curator: {{ entry.figure_description.comment }}
{% endif %}
```

Is this not good enough actually? If `comment` is `None` this is false and if the comment is the empty string it is also false.

Reply:
Yeas right.